pulumi/folio/index.ts

```diff
@@ -0,0 +1,203 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import * as awsx from "@pulumi/awsx";
+import * as eks from "@pulumi/eks";
+
+// Set some default tags which we will add to when defining resources.
+const tags = {
+    "Owner": "CTA",
+    "Environment": pulumi.getStack(),
+    "Product": "FOLIO",
+    "Accounting": "cubl-folio",
+    "DataClassificationCompliance": "standard"
+};
+
+// Allocate a new VPC with the default settings using pulumi Crosswalk.
+const vpcName = "folio-vpc";
+const vpc = new awsx.ec2.Vpc(vpcName, {
+    tags: { "Name": vpcName, ...tags },
+
+    // If we have too many AZs and NAT gateways we'll run out of EIPs with our current quota. 
+    numberOfAvailabilityZones: 2,
+
+    // We could have these be equal to the number of availability zones for greater
+    // fault tolerance. Although the cost will be higher.
+    numberOfNatGateways: 1,
+
+    // See https://github.com/pulumi/pulumi-eks/blob/master/examples/subnet-tags/index.ts
```

Comment:
Updating to permalink
```suggestion
    // See https://github.com/pulumi/pulumi-eks/blob/e03751425604b5193c00cd2f8858d9b16f7901f0/examples/subnet-tags/index.ts
```